### ISSUE TITLE
[Usable] Inside controls(mute,play etc..) of the Audio/Video clips are not accessible while navigate in scan mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1315]
**Actual Result:**
Inside controls(mute,play etc..) of the Audio/Video clips are not accessible while navigate in scan mode.

Expected Result:
Inside controls(mute,play etc..) of the Audio/Video clips should be accessible while navigate in scan mode.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'accessibility' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Page will upload.
5. Open Narrator with scan mode (Caps+Space)
6. Navigate to Audio/Video clip by using Up/Down arrow keys.
7. Observe that "mute,play,full screen etc.." controls of audio/video clips are not accessible in scan mode.

**User Impact:**
Users who rely on screen readers may get confused and face difficulty if navigation is not correct or logical and controls are not accessible in scan mode.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version**: 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_audio clip controls are not accessible in scan mode.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547765/Usable_audio.clip.controls.are.not.accessible.in.scan.mode.zip)

**Comment by corinagum** [comment]
Confirmed as external issue by accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable

**Comment by pgopinadhu** [comment]
#A11yUsable;#A11ySev4;
